common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -899,6 +905,90 @@ public static ClassLoader getSystemClassLoader() {
+                            if (line.startsWith("cache_alignment")) {
+                                String[] parts = line.split(":");
+                                return Integer.parseInt(parts[1].trim());
+                            }
+                        }
+                    } catch (Exception cause) {
+                        logger.debug("Failed to retrieve cache line size", cause);
+                        return -1;
+                    } finally {
+                        if (in != null) {
+                            try {
+                                in.close();
+                            } catch (IOException ignore) {
+                                // ignore on close
+                            }
+                        }
+                    }
+                } else if (IS_OSX) {
+                    Process p = null;
+                    BufferedReader in = null;
+                    try {
+                        p = Runtime.getRuntime().exec(new String[] { "sysctl", "hw.cachelinesize" });
+                        in = new BufferedReader(new InputStreamReader(p.getInputStream(), CharsetUtil.US_ASCII));
+                        String line = in.readLine();
+                        in.close();
+                        try {
+                            int exitCode = p.waitFor();
+                            if (exitCode != 0) {
+                                logger.debug("Failed to retrieve cache line size," +
+                                        " 'sysctl hw.cachelinesize' did exit with code " + exitCode);
+                                return -1;
+                            }
+                        } catch (InterruptedException e) {
+                            Thread.currentThread().interrupt();
+                        }
+                        if (line != null) {
+                            String[] parts = line.split(":");
+                            return Integer.parseInt(parts[1].trim());
+                        }
+                    } catch (Exception e) {
+                        // Failed to run the command.
+                        logger.debug("Failed to retrieve cache line size", e);
+                        return -1;
+                    } finally {
+                        if (in != null) {
+                            try {
+                                in.close();
+                            } catch (IOException ignore) {
+                                // Ignore
+                            }
+                        }
+                        if (p != null) {
+                            try {
+                                p.destroy();
+                            } catch (Exception ignore) {
+                                // Just ignore
+                            }
+                        }
+                    }
+                }
+                logger.debug("Failed to retrieve cache line size");
```

Comment:
@kiril-me it's question for me?
I mean what it will be printed on the Windows or other than linux/osx systems. But it's not a failure.